### PR TITLE
Added the documentation for a plugin license check

### PR DESCRIPTION
**Pull request by ngoedde**

### CODE REVIEW

File: source/developers-guide/index.html [hunk 1]
```diff
@@ -15,5 +15,6 @@
     <li><a href="{{ site.url }}/developers-guide/shopware-5-performance-for-devs/">Shopware 5 performance guide for developers</a></li>
     <li><a href="{{ site.url }}/developers-guide/password-encoder/">Shopware password encoder</a></li>
     <li><a href="{{ site.url }}/developers-guide/debugging/">Debugging shopware</a></li>
+    <li><a href="{{ site.url }}/developers-guide/shopware-5-plugin-license/">Shopware 5 license check in plugins</a></li>
```
Comment (bcremer) [inline source/developers-guide/index.html]: Please get rid of shopware-5 in the filename and so on. It's just "License check in plugins"

File: source/developers-guide/shopware-5-plugin-license/index.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+If you have ordered the Shopware license check for your plugin, you will find the automatically generated code snippet in your account, that you need to include in your plugin's Bootstrap.php.
+This document describes the basic procedure.
+
+
+
+## The Snippet
+
+The generated code-snippet usually looks like this:
+
+```php
```
Comment (bcremer) [inline source/developers-guide/shopware-5-plugin-license/index.md]: Do not use code fences "```" and indentation. Just use the code fences.

File: source/developers-guide/shopware-5-plugin-license/index.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+throw an exception which cancels the execution of the request. If *"throwException"* is set to false the method simply returns a boolean which indicates whether the license check was successful or not.
+Usually exceptions should be thrown during *installation*, *update* or when executing backend operations, to confront the user about the problem. Never throw exceptions in the frontend because this may
+cripple your shop.
+
+## Example
+
+```php
+<?php
+
+final class Shopware_Plugins_Frontend_SwagMyPlugin_Bootstrap extends Shopware_Components_Plugin_Bootstrap
```
Comment (bcremer) [inline source/developers-guide/shopware-5-plugin-license/index.md]: get rid of the final keyword here...

**Comment by bcremer** [comment]
@ngoedde added comment regarding the filename. please also squash the commits together afterwards.

**Comment by tiagojsag** [comment]
https://github.com/shopware/devdocs/pull/113

**Comment by Haehnchen** [comment]
please deploy this. updated plugin on the way... :)

**Pull request closed without merging**